ci/owncloud/upload_to_owncloud.py

```diff
@@ -0,0 +1,3 @@
+from webdav3.client import Client
+from opts import debug, message, opts
```

Comment:
https://stackoverflow.com/questions/7505988/importing-from-a-relative-path-in-python
No need for a soft link.

Reply:
From some reading I understand that implementing it the proper way would require adjustments to the filesystem (adding `__init__.py` files so that we have a package system, which is also kind of wrong, since what we're using is not package).

Changing the import to `from ...tests.opts import debug, message, opts` will result in an `ImportError` since there is no parent package.

There's a hacky way of doing it but it is highly discouraged by PEP.

A good solution IMO would be to implement a package of python utilities inside the prplMesh repository and accessing it with relative imports. But as it stands now, I would say using a symlink is the involves creating fewer files.